libraries/chain/include/eosio/chain/log_catalog.hpp

```diff
@@ -0,0 +1,292 @@
+#pragma once
```

Comment:
Should those 3 log_* files belong to `libraries/state_history/include/eosio/state_history`?

Reply:
Sorry, just saw those 3 files are going to be used by https://github.com/AntelopeIO/leap/pull/532 too. Then they do belong to chain include directory.